dev/com.ibm.ws.threading/src/com/ibm/ws/threading/internal/ThreadPoolController.java

```diff
@@ -395,13 +413,19 @@
     private double processCpuUtil = -1.0;
     private double systemCpuUtil = -1.0;
     private double cpuUtil = -1.0;
+    private static DecimalFormat df = new DecimalFormat("0.00", DecimalFormatSymbols.getInstance(Locale.US));
 
     /**
      * The controller uses the threadpool queue depth as an input to some of the
      * decisions it makes.
      */
     private int queueDepth = 0;
 
+    /**
+     * How many threads are active (running tasks) at the current controller cycle
+     */
+    private int activeThreads = 0;
+
```

Comment:
It would appear that this does not need to be an instance scoped variable, but instead can be passed to toIntervalData in evaluateInterval.  It is only used in those two methods.

Reply:
I've just added a use of activeThreads in adjustPoolSize ... could still change it to be passed into the methods as a parameter, not sure how much difference it makes?